SparkPost/2web2ui
# SA-1758: Metrics changed to be in expandable

### What Changed

- Metrics are now in drawer
- Metrics are now listed in alphabetical order vertically

### How To Test
- Go to report builder
- Verifyt hat metrics drawer form works correctly
- Verify that metrics are separated in expandables
- Verify that the changes match mocks

### To Do
